omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/wrapper/openmrs1_8/CohortMember1_8.java

```diff
@@ -22,7 +22,7 @@
  */
 public class CohortMember1_8 implements Serializable {
 	
-	public static final long serialVersionUID = 1L;
+	public static final long SERIAL_VERSION_UID = 1L;
```

Comment:
Btw, you should not use uppercase for `serialVersionUID`. Even though constants should be capitalize, there is an exception for the variable `serialVersionUID`, which is a special variable that is used by the serialization mechanism. Same applied for `serialPersistentFields` valriable.